byway/src/main/webapp/base.js

```diff
@@ -12,30 +12,30 @@ function setProgressBar(pageNumber){
 }
 
 /**
- * Go through url to retrieve the trip id.
- * @returns String of trip ID
+ * Go through url to retrieve the trip key.
+ * @return String of trip key
  */
-function getTripIdFromUrl() {
-  return new URLSearchParams(location.search).get('tripId');
+function getTripKeyFromUrl() {
+  return new URLSearchParams(location.search).get('tripKey');
 }
 
 /**
- * Sets the trip Id passed in to the next page, specified
- * by the nextPagePath. If the trip Id is null, redirects the
+ * Sets the trip key passed in to the next page, specified
+ * by the nextPagePath. If the trip key is null, redirects the
  * user to the page where they can make a new trip and have
- * a valid trip Id to reference.
+ * a valid trip key to reference.
  * Note: Requires an element in the document with id "next-button"
  * with an href attribute, e.g. an 'a' tag.
- * @param {String} tripId unique value for a trip
+ * @param {String} tripKey unique value for a trip
  * @param {String} nextPagePath HTTP path for next page
  */
-function configureTripIdForNextPage(tripId, nextPagePath) {
-  if(tripId === null) {
+function configureTripKeyForNextPage(tripKey, nextPagePath) {
+  if(tripKey === null) {
     // Send back to page where a trip can be made
-    alert("tripId not created! Create a new trip.");
+    alert("tripKey not created! Create a new trip.");
     window.location.href = '/index.html';
   } else {
     let nextPage = document.getElementById("next-button");
```

Comment:
maybe you shouldn't include the let nextPage=document.getElementbyId("next-button") if we are all going to be using this function since we might name it something different. I would recommend just returning url

Reply:
Updated to be more general and fill any path with the tripKey param